docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -539,9 +539,21 @@ For details on how online index backfill works, see the [Online Index Backfill](
 
 Default: `false`
 
+##### --ysql_sequence_cache_method
+
+Specifies where to cache sequence values.
+
+Valid values are `connection` and `server`.
+
+This flag requires the YB-TServer `yb_enable_sequence_pushdown` flag to be true (the default). Otherwise, the default behavior will occur regardless of this flag's value.
```

Comment:
We don't have docs for `yb_enable_sequence_pushdown` ? Asking to link it.

Reply:
No we don't. There is a precedent - the other flag that begins with "yb_enable", which is referred to several times but not actually documented